src/main/java/maksym/fedorenko/bookstore/dto/order/UpdateOrderRequestDto.java

```diff
@@ -0,0 +1,7 @@
+package maksym.fedorenko.bookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import maksym.fedorenko.bookstore.model.Order;
+
+public record UpdateOrderRequestDto(@NotNull Order.Status status) {
```

Comment:
minor

you migth change the import to simplify type:
```suggestion
public record UpdateOrderRequestDto(@NotNull Status status) {
```